lib/client.js

```diff
@@ -128,6 +128,17 @@ class Client {
     return this._get(helper.taskPaths.deleteTask(taskId))
   }
 
+  /**
+   * Get the structure of one table.
+   * NOTE: This endpoint is not available in production yet.
+   *
+   * @param {String} tableName Table.
+   * @returns {Object} A promise with the result.
+   */
+  table(tableName) {
+    return this._get(`/table/${tableName}`);
```

Comment:
Does this function return an object, and with what structure?

Reply:
Returns an object that contains serrea response where 'object' field it's an array that contains columns field name and type.

Reply:
Do you think it is worth it returning just the `object` field then?

Reply:
I think client should be uniform so if `query` method returns serrea's response, `table` should do the same. Otherwise Serrea has other important fields like `cid` that is the unique id for request. This code allows trace the requests in all internal systems.
So I consider sdk should returns whole response.

Are you agree with me?

Reply:
OK, thanks!